### DISCUSSION TITLE
Allure Playwright: outputFolder in playwright.config.ts gets ignored

### DISCUSSION THREAD

**Discussion by nreichstein** [allure-framework/allure2#2649] [Bug Reports] [answered]
Hey, 
I'm using allure-playwright (v3.0.0-beta.7) and I'm trying to set a different "outputFolder" than the default one.
The reporter in the playwright.config.ts looks like this:
![image](https://github.com/user-attachments/assets/4a472c3b-6c79-4867-9b68-dc218b8badb1)
The issue is that it always creates a folder with the default name "allure-results" in the root directory no matter how I adjust the configuration.
However other configuration parameters like ''suiteTitle" are working.

**Comment by baev** [comment, marked as answer]
Since 3.0.0, `outputFolder` was renamed to `resultsDir` across all the integrations. Readme & Docs will be updated before the final release

**Comment by miranda-gv** [comment]
this is how it worked for me if anyone is having the same issue.

```
const allureDir = path.resolve(__dirname, 'test-reports', 'allure-results');

reporter: [
    ['allure-playwright', { resultsDir: allureDir}],
    }]
```

Also when you generate reports the new location should be added in the command like this.

`npx allure generate test-reports/allure-results --clean -o test-reports/allure-report && npx allure open test-reports/allure-report`
